config/routes.rb

```diff
@@ -10,13 +10,27 @@
   get 'public/:id' => 'public_documents#show'
   get 'review/:id' => 'public_documents#show'
 
-  devise_for :users, controllers: {registrations: 'registrations', omniauth_callbacks: 'omniauth_callbacks'}
+  # devise_for :users, controllers: {registrations: 'registrations', omniauth_callbacks: 'omniauth_callbacks'}
 
+  devise_for :users, skip: :saml_authenticatable, controllers: {
+               registrations: 'registrations',
+               omniauth_callbacks: 'omniauth_callbacks',
+               sessions: 'user/sessions',
+               passwords: 'user/passwords'
+             }
   devise_for :admin_users, ActiveAdmin::Devise.config
+  # devise_scope :user do
+  #   match '/users/auth/wordpress_hosted/setup' => 'omniauth_callbacks#setup', via: :all, as: 'wordpress_host_setup'
+  # end
```

Comment:
We need to be able to toggle between OAuth and SAML -- this appears to remove support for OAuth callbacks?